Library/Formula/telegraf.rb

```diff
@@ -77,5 +77,8 @@ def plist; <<-EOS.undent
 
   test do
     assert_match /Telegraf/, shell_output("telegraf -version")
+    system("telegraf -sample-config > #{testpath}/config.toml")
```

Comment:
Please use `shell_output` and `(testpath/"config.toml").write` to write out the file, thanks!